src/cr/cube/matrix.py

```diff
@@ -516,6 +682,12 @@ def _row_generator(self):
 
     @lazyproperty
     def _rows_margin(self):
+        """1D np.float/int64 ndarray of weighted-N for each matrix row.
+
+        These "opposing-margin" values are required for vector-level calculations but
+        must be calculated at matrix level. The values are `np.int64` when the source
```

Comment:
please add the part about calculation that is "cross-vector", similar to the one above. it's priceless.